src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/manageCaseAssignments/casesharing.scala

```diff
@@ -0,0 +1,259 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios.manageCaseAssignments
+
+import com.typesafe.config._
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.corecasedata.scenarios.utils.Environment
+
+object casesharing {
+
+  val config: Config = ConfigFactory.load()
+  val xuiClientId = "xuiwebapp"
+  val xuiScope = "openid profile roles manage-user create-user manage-roles search-user"
+  val ccdClientId = "ccd_gateway"
+  val ccdScope = "openid profile authorities acr roles openid profile roles"
+  val feedCSUserDataLarge = csv("CaseSharingUsers_Large.csv").circular
+  val feedCSUserDataSmall = csv("CaseSharingUsers_Small.csv").circular
+  val mcaUrl = "http://aac-manage-case-assignment-perftest.service.core-compute-perftest.internal"
+
+  val headers_0 = Map( //Authorization token needs to be generated with idam login
+    "Authorization" -> "AdminApiAuthToken ",
+    "Content-Type" -> "application/json")
+
+  val xuiwebappClientSecret = config.getString("xuiwebappCS")
+  val ccdGatewayClientSecret = config.getString("auth.ccdGatewayCS")
+
+  val CDSGetRequestLarge =
+
+    feed(feedCSUserDataLarge)
+
+      //    .exec {
+      //        session =>
+      //          println(session("userEmail").as[String])
+      //          session
+      //      }
+
+      .exec(http("01_Create_GetS2SToken")
```

Comment:
There's already a common s2s function defined that you could reuse here

Reply:
this code is existing in the repo and not mine - the casesharing.scala file has only been moved into a new folder.

Reply:
I have now removed this file from the repo as the code is being rewritten as part of the manage case PI28